src/shared/proto/primitive.proto

```diff
@@ -9,31 +9,15 @@ message Primitive
     oneof primitive
     {
         EstopPrimitive estop                  = 1;
-        ChipPrimitive chip                    = 2;
-        KickPrimitive kick                    = 3;
-        MovePrimitive move                    = 4;
-        SpinningMovePrimitive spinning_move   = 5;
-        StopPrimitive stop                    = 6;
-        DirectControlPrimitive direct_control = 7;
+        MovePrimitive move                    = 2;
+        SpinningMovePrimitive spinning_move   = 3;
```

Comment:
Is there a point in keeping this primitive in proto?

Reply:
nope, but it will be removed as part of Jons PR